.github/ISSUE_TEMPLATE/A.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Having trouble getting started?
+about: Please contact us at support@bugsnag.com for assistance with integrating Bugsnag
+  into your application.
```

Comment:
Do we need to update your script to apply the right capitalisation?

```suggestion
about: Please contact us at support@bugsnag.com for assistance with integrating BugSnag into your application.
```

Reply:
We could do, although the statement is true for integrating any BugSnag library into your application :)